README.md

```diff
@@ -105,6 +105,14 @@ Please check [Tips for using Azure CLI effectively](https://docs.microsoft.com/e
 
 For more usage examples, take a look at our [GitHub samples repo](http://github.com/Azure/azure-cli-samples) or [https://docs.microsoft.com/cli/azure/overview](https://docs.microsoft.com/cli/azure/overview).
 
+## Data Collection
```

Comment:
```suggestion
## Telemetry
```

Reply:
The `Data Collection` title follows the [instruction](https://docs.opensource.microsoft.com/content/releasing/telemetry.html)

Reply:
lesson learnt. Telemetry is used widely in vscode extensiosn repos,  learnt from there :)